content/en/docs/instrumentation/java/automatic/agent-config.md

```diff
@@ -69,8 +69,189 @@ See [common instrumentation configuration properties][common-config].
+| Restlet                                          | restlet                   |
+| Scala ForkJoinPool                               | scala-fork-join           |
+| Spark Web Framework                              | spark                     |
+| Spring Batch                                     | spring-batch              |
+| Spring Core                                      | spring-core               |
+| Spring Data                                      | spring-data               |
+| Spring Integration                               | spring-integration        |
+| Spring Kafka                                     | spring-kafka              |
+| Spring RabbitMQ                                  | spring-rabbit             |
+| Spring RMI                                       | spring-rmi                |
+| Spring Scheduling                                | spring-scheduling         |
+| Spring Web                                       | spring-web                |
+| Spring WebFlux                                   | spring-webflux            |
+| Spring Web MVC                                   | spring-webmvc             |
+| Spring Web Services                              | spring-ws                 |
+| Spymemcached                                     | spymemcached              |
+| Twilio SDK                                       | twilio                    |
+| Twitter Finatra                                  | finatra                   |
+| Undertow                                         | undertow                  |
+| Vaadin                                           | vaadin                    |
+
+**Note:** When using environment variables, dashes (`-`) should be converted to
+underscores (`_`). For example, to suppress traces from `akka-actor` library,
+set `OTEL_INSTRUMENTATION_AKKA_ACTOR_ENABLED` to `false`.
+
+#### Even more fine-grained control
+
+You can also exclude specific classes from being instrumented.
+
+This can be useful to completely silence spans from a given class/package.
+
+Or as a quick workaround for an instrumentation bug, when byte code in one
+specific class is problematic.
+
+This option should not be used lightly, as it can leave some instrumentation
+partially applied, which could have unknown side-effects.
+
+If you find yourself needing to use this, it would be great if you could drop us
+an issue explaining why, so that we can try to come up with a better solution to
+address your need.
+
+| System property                | Environment variable           | Purpose                                                                                           |
+|--------------------------------|--------------------------------|---------------------------------------------------------------------------------------------------|
+| otel.javaagent.exclude-classes | OTEL_JAVAAGENT_EXCLUDE_CLASSES | Suppresses all instrumentation for specific classes, format is "my.package.MyClass,my.package2.*" |
+
```

Comment:
@open-telemetry/java-instrumentation-approvers do you know of anyone using this? it's a pretty advanced and dangerous setting, so my initial though is to remove it from this page and find another less prominent location for it

Reply:
> my initial though is to remove it from this page and find another less prominent location for it

👍 

I remember we had a couple issues when people actually tried using it and it didn't work as expected. Maybe we can keep this one in the instrumentation repo? In some advanced/internal config options doc.

Reply:
thx @mateuszrzeszutek. @chalin let's go ahead and remove this section, I will find another location for it in the instrumentation repo

Reply:
Section removed.